test/core/br.wast

```diff
@@ -442,25 +442,6 @@
   "type mismatch"
 )
 
-(assert_invalid
-  (module
-    (func $type-operand-missing
-      (block (result i32) (br 0))
-      (i32.eqz) (drop)
-    )
-  )
-  "type mismatch"
-)
-(assert_invalid
-  (module
-    (func $type-operand-missing-in-block
-      (i32.const 0)
-      (block (result i32) (br 0))
-      (i32.eqz) (drop)
-    )
-  )
-  "type mismatch"
-)
 (assert_invalid
   (module
     (func $type-operand-missing-in-if
```

Comment:
I believe the this and the next are duplicates  of `$type-{then,else}-break-empty-vs-num` in if.wast, so can be removed as well.

Reply:
Indeed, I will remove them as well.